src/nvc_mount.c

```diff
@@ -49,12 +50,9 @@ static char *
 mount_directory(struct error *err, const char *root, const struct nvc_container *cnt, const char *dir)
 {
         char src[PATH_MAX];
-        char dst[PATH_MAX];
         if (path_join(err, src, root, dir) < 0)
                 return (NULL);
```

Comment:
Consider changing this to `path_resolve_full()` to be symmetrical with what we now do in `mount_firmware()`.